src/zenml/integrations/kserve/steps/kserve_deployer.py

```diff
@@ -163,6 +169,37 @@ def torch_config_validate(cls, v: Optional[str]) -> Optional[str]:
         return v
 
 
+class CustomDeployParameters(BaseModel):
+    """Custom model deployer step extra parameters.
+
+    Attributes:
+        predict_function: Path to Python file containing predict function.
+    """
+
+    predict_function: str
+
+    @validator("predict_function")
+    def predict_function_validate(cls, predict_func_path: str) -> str:
+        """Validate predict function.
+
+        Args:
+            predict_func_path: predict function path
+
+        Returns:
+            predict function path
+
+        Raises:
+            ValueError: if predict function path is not valid
+        """
+        if not predict_func_path:
```

Comment:
I think pydantic already covers this check as this is a required, non-optional string attribute